### PR TITLE
Content layout

### PR DESCRIPTION
**Pull request by tammaroivan**
##  Summary

- Content Layout (Front-End & Back-End)

## Screenshot
image

image

## Trello Cards

https://trello.com/c/hwrK1kXb/23-content-layout

### CODE REVIEW

File: maquetado/index.html [hunk 1]
```diff
@@ -10,16 +10,72 @@
...
-	
-	<div class="cards">
-		<div class="tech">
-			<img src="assets/react_icon.svg" alt="">
-			<p>ReactJS</p>
-			<p><strong>Release: 2013</strong></p>
-			<p class="description">React is declarative, component-base JavaScript library for building user interfaces.</p>
-		</div>
-	</div>
+
```
Comment (FrankIglesias) [inline maquetado/index.html]: delete line

File: maquetado/index.html [hunk 2]
```diff
@@ -10,16 +10,72 @@
...
+  <div class="main">
+    <h1 class="title">Front-End Tools</h1>
+
```
Comment (FrankIglesias) [inline maquetado/index.html]: same

File: maquetado/index.html [hunk 3]
```diff
@@ -10,16 +10,72 @@
...
+  	<div class="cards">
+  		<div class="tech">
+  			<img src="assets/react_icon.svg" alt="">
+  			<p>ReactJS</p>
+  			<p><strong>Release: 2013</strong></p>
```
Comment (FrankIglesias) [inline maquetado/index.html]: add a class to this font weight

File: maquetado/index.html [hunk 4]
```diff
@@ -10,16 +10,72 @@
...
+  			<p class="description">React is declarative, component-base JavaScript library for building user interfaces.</p>
+  		</div>
+      <div class="tech">
+  			<img src="assets/angular_icon.svg" alt="">
```
Comment (FrankIglesias) [inline maquetado/index.html]: add an alt

File: maquetado/index.html [hunk 5]
```diff
@@ -10,16 +10,72 @@
...
+  			<p>Angular JS</p>
+  			<p><strong>Release: 2010</strong></p>
+  			<p class="description">AngularJS (commonly referred to as AngularJS 1.X) is a JavaScript framework for building SPAs with an MVC architechture.</p>
+  		</div>
+      <div class="tech">
+  			<img src="assets/vue_icon.svg" alt="">
```
Comment (FrankIglesias) [inline maquetado/index.html]: same

File: maquetado/index.html [hunk 6]
```diff
@@ -10,16 +10,72 @@
...
+  			<p>Vue.js</p>
+  			<p><strong>Release: 2014</strong></p>
```
Comment (FrankIglesias) [inline maquetado/index.html]: same

File: maquetado/index.html [hunk 7]
```diff
@@ -10,16 +10,72 @@
...
+  			<p class="description">VueJS (pronounced /vjuː/, like view) is an open-source progressive JavaScript framework for building user interfaces.</p>
+  		</div>
+      <div class="tech">
+  			<img src="assets/elm_icon.svg" alt="">
```
Comment (FrankIglesias) [inline maquetado/index.html]: same

File: maquetado/index.html [hunk 8]
```diff
@@ -10,16 +10,72 @@
...
+  			<p class="description">Elm is a domain-specific functional programming language for declaratively creating web browser-based graphical user interfaces.</p>
+  		</div>
+      <div class="tech">
+  			<img src="assets/polymer_icon.svg" alt="">
+  			<p>Polymer</p>
+  			<p><strong>Release: 2015</strong></p>
+  			<p class="description">Polymer is an open-source JavaScript library for building web applications using Web Components.</p>
+  		</div>
+  	</div>
+
```
Comment (FrankIglesias) [inline maquetado/index.html]: same

File: maquetado/index.html [hunk 9]
```diff
@@ -10,16 +10,72 @@
...
+    <h1 class="title">Back-End Tools</h1>
+
```
Comment (FrankIglesias) [inline maquetado/index.html]: same

File: maquetado/scss/styles.scss [hunk 10]
```diff
@@ -8,8 +8,14 @@ body {
 	padding: 100px;
 }
 
+.main {
+	width: 70%;
```
Comment (FrankIglesias) [inline maquetado/scss/styles.scss]: use a padding instead of a with with percentage

File: maquetado/scss/styles.scss [hunk 11]
```diff
@@ -8,8 +8,14 @@ body {
...
+	margin: 0 auto;
+}
+
 .cards {
 	display: flex;
 	flex-wrap: wrap;
-	margin: 50px 0;
+	margin: 50px;
```
Comment (FrankIglesias) [inline maquetado/scss/styles.scss]: use shorthand for this

File: maquetado/scss/tech.scss [hunk 12]
```diff
@@ -8,12 +8,14 @@
 	text-align: center;
 	padding: 15px;
 	position: relative;
-	width: 200px;
-	height: 300px;
+	width: 210px;
+	height: 330px;
```
Comment (FrankIglesias) [inline maquetado/scss/tech.scss]: use flexbox for this height

File: maquetado/scss/tech.scss [hunk 13]
```diff
@@ -8,12 +8,14 @@
...
+	margin: 25px;
 
 	img {
-		width: 125px;
+		width: 100px;
 		margin: 5px 0;
+		max-height: 100px;
```
Comment (FrankIglesias) [inline maquetado/scss/tech.scss]: set a height as auto or delete the width and use a height

**Review by FrankIglesias — CHANGES_REQUESTED** [review]
Check indentation

**Comment by tammaroivan** [comment]
New fixes

File: maquetado/index.html [hunk 14]
```diff
@@ -10,16 +10,68 @@
...
-			<p>ReactJS</p>
-			<p><strong>Release: 2013</strong></p>
-			<p class="description">React is declarative, component-base JavaScript library for building user interfaces.</p>
-		</div>
-	</div>
 
+  <div class="main">
+    <h1 class="title">Front-End Tools</h1>
+  	<div class="cards">
+  		<div class="tech">
```
Comment (FrankIglesias) [inline maquetado/index.html]: check indentation

File: maquetado/scss/styles.scss [hunk 15]
```diff
@@ -8,8 +8,17 @@ body {
 	padding: 100px;
 }
 
+.main {
```
Comment (FrankIglesias) [inline maquetado/scss/styles.scss]: same

File: maquetado/scss/styles.scss [hunk 16]
```diff
@@ -8,8 +8,17 @@ body {
...
+	padding: 0 15%;
```
Comment (FrankIglesias) [inline maquetado/scss/styles.scss]: use px instead

File: maquetado/scss/styles.scss [hunk 17]
```diff
@@ -8,8 +8,17 @@ body {
...
+	margin: 0 auto;
```
Comment (FrankIglesias) [inline maquetado/scss/styles.scss]: why this?

File: maquetado/scss/styles.scss [hunk 18]
```diff
@@ -8,8 +8,17 @@ body {
...
+
 .cards {
 	display: flex;
 	flex-wrap: wrap;
-	margin: 50px 0;
+	margin: 20px 50px 50px 50px;
+}
+
+.bold {
+  font-weight: 600;
```
Comment (FrankIglesias) [inline maquetado/scss/styles.scss]: font-weight 500?

File: maquetado/scss/tech.scss [hunk 19]
```diff
@@ -8,12 +8,14 @@
 	text-align: center;
 	padding: 15px;
 	position: relative;
-	width: 200px;
-	height: 300px;
+	width: 210px;
```
Comment (FrankIglesias) [inline maquetado/scss/tech.scss]: check indentation

File: maquetado/scss/tech.scss [hunk 20]
```diff
@@ -8,12 +8,14 @@
...
+	height: 330px;
+	margin: 25px;
 
 	img {
```
Comment (FrankIglesias) [inline maquetado/scss/tech.scss]: avoid using tags in scss

File: maquetado/scss/tech.scss [hunk 21]
```diff
@@ -8,12 +8,14 @@
...
-		width: 125px;
+		width: 100px;
 		margin: 5px 0;
+		max-height: auto;
```
Comment (FrankIglesias) [inline maquetado/scss/tech.scss]: use height instead

**Review by FrankIglesias — CHANGES_REQUESTED** [review]

File: maquetado/scss/styles.scss [hunk 22]
```diff
@@ -1,15 +1,23 @@
 $green: #BED23C;
 
 * {
-	font-family: 'Roboto', sans-serif;
+  font-family: 'Roboto', sans-serif;
 }
 
 body {
-	padding: 100px;
+  padding: 100px;
```
Comment (FrankIglesias) [inline maquetado/scss/styles.scss]: remove this, add a margin to the .main class

File: maquetado/scss/styles.scss [hunk 23]
```diff
@@ -1,15 +1,23 @@
...
 .cards {
-	display: flex;
-	flex-wrap: wrap;
-	margin: 50px 0;
+  display: flex;
+  flex-wrap: wrap;
+  margin: 20px 50px 50px 50px;
+}
+
+.bold {
```
Comment (FrankIglesias) [inline maquetado/scss/styles.scss]: move this to a `font.scss` file

**Review by FrankIglesias — CHANGES_REQUESTED** [review]

**Comment by FrankIglesias** [comment]
resolve conflicts

File: maquetado/scss/styles.scss (hunk 23 above)
Comment (tammaroivan) [reply maquetado/scss/styles.scss]: class .bold moved to scss/font.scss

**Review by FrankIglesias — APPROVED** [review]

**Pull request merged**